docs/source/rule-staging.rst

```diff
@@ -0,0 +1,109 @@
+Rule Staging
+============
+
+Rule staging allows dynamic toggling of rules from a staging to production state, and vice versa.
+Staged rules will only send alerts for historical retention, and the alerts will not be sent to any
+user-defined outputs, such as Slack, PagerDuty, etc.
```

Comment:
Can you add some context about why this is a useful feature? E.g. "To help ensure new rules do not flood your outputs with false positives, rules can be _staged_..."